pkg/skaffold/errors/errors_test.go

```diff
@@ -243,6 +243,22 @@ func TestShowAIError(t *testing.T) {
 				Suggestions: reportIssueSuggestion(config.SkaffoldOptions{}),
 			},
 		},
+		{
+			description: "deploy failed",
+			opts:        config.SkaffoldOptions{},
```

Comment:
you need to pass in different contexts in `context` field, 
1. Minikube
2. Non Local cluster or GKE

The config.ContextConfig object has a `Kubecontext` 
1. Setting the value to "minikube" will test  local minikube scenario
2. Setting the value to something other than "minikube" will test the remote cluster scenario.